### ISSUE TITLE
createClient fails

### ISSUE THREAD

**Issue by CloudGlueIO** [vpulim/node-soap#342]
https://api.demo.ezidebit.com.au/v3-3/nonpci?singleWsdl

Already reported need to not assert if encounters an unknown child e.g. wsp:Policy

-> Remove 'assert(false, "Invalid child type");' from DefinitionsElement.prototype.addChild

Once I comment out the above, error ends up being thrown trying to reference property of null object in 'MessageElement.prototype.postProcess'

```
    this.element = definitions.schemas[definitions.xmlns[ns]].elements[nsName.name];
    this.element.targetNSAlias = ns;
```

In this case the xmlns:tns definition is being overwritten as it is redefined throughout the wsdl.  I have no idea is this is allowed behaviour.

Basically if you comment out the assert above and run against the URL above the error will occur.  WSDL is such a pig.

This issue relates to AddCustomer and NewCustomer.
